### ISSUE TITLE
Limit which instances are collected

### ISSUE THREAD

**Issue by cgrinds** [NetApp/harvest#868]
### Discussed in https://github.com/NetApp/harvest/discussions/867

<div type='discussions-op-text'>

<sup>Originally posted by **luddite516** February 17, 2022</sup>
We have a use case where we are collecting file lock statistics for a specific vserver.  I have built the following template to collect lock stats:

name:               Locks
query:              lock-get-iter
object:             lock

collect_only_labels: true

counters:
  lock-info:
    - ^node
    - ^volume
    - ^vserver
    - ^path
    - ^^lockid
    - ^client-address => client_ip
    - ^smb-open-type  => smb_open_type

export_options:
  instance_keys:
    - lockid
  instance_labels:
    - vserver
    - volume
    - node
    - path
    - client_ip
    - smb_open_type

The above template collects all the locks in the cluster, and there may be hundreds of thousands of them.  

Is there any way to limit which instances are collected, to reduce load on cluster?  For example, limit collection to only a specific vserver or volume?

Thanks.</div>

**Comment by cgrinds** [comment]
As of 24.02, filtering is supported by the Rest and ZapiPerf collectors

**Comment by luddite516** [comment]
Yes, I believe we could move to REST.  We are going to have to get there
eventually.
